README.md

```diff
@@ -1,17 +1,25 @@
-# The Algorithms - PHP
+#FOR EDUCATIONAL PURPOSE:
 
-### All algorithms implemented in Php (for education)
+#The Algorithms - PHP
 
-These implementations are for learning purposes. They may be less efficient than the implementations in the Php standard library.
+### All algorithms implemented in Php
```

Comment:
```suggestion
### All algorithms implemented in Python - for education
```

I'd suggest following what's been done in the other `TheAlgorithms` repos, e.g. [Python](https://github.com/TheAlgorithms/Python/blob/master/README.md#all-algorithms-implemented-in-python---for-education)